commands/audit/sca/java/gradle.go

```diff
@@ -57,10 +57,22 @@ var gradleDepTreeJar []byte
 
 type gradleDepTreeManager struct {
 	DepTreeManager
+	isCurationCmd bool
+}
+
+func NewGradleDepTreeManager(params *DepTreeParams, cmdName MavenDepTreeCmd) *gradleDepTreeManager {
```

Comment:
`cmdName MavenDepTreeCmd` ?
is there a new command to the gradle dep tree? why is this added? if so please move the type to `deptreemanager`